lib/dealership.rb

```diff
@@ -0,0 +1,57 @@
+require 'active_support'
+require 'active_support/core_ext/numeric/conversions'
+require 'pry'
+
+
+class Dealership
+  attr_reader :name, :address, :inventory
+
+  def initialize(name, address)
+    @name = name
+    @address = address
+    @inventory = []
+  end
+
+  def add_car(car)
+    @inventory << car
+  end
+
+  def inventory_count
+    @inventory.count
+  end
+
+  def has_inventory?
+    return true if @inventory.count > 0
+    false
+  end
+
+  def cars_by_make(make)
+    @inventory.select { |car| car if car.make == make }
+  end
+
+  def total_value
+    total = 0
+    @inventory.each { |car| total += car.total_cost}
+    total
+  end
+
+  def details
+    @deets = { "total_value" => total_value,
+               "address" => @address}
+  end
+
+  def average_price_of_car
+    average = total_value / inventory_count
+    average.to_s(:delimited)
```

Comment:
Well done. Where did you find `:delimited`?

Reply:
[StackOverflow!](https://stackoverflow.com/questions/1078347/rails-is-there-a-rails-trick-to-adding-commas-to-large-numbers) I googled it because I figured there had to be a way to return a large number with a comma.